util/json.js

```diff
@@ -0,0 +1,9 @@
+import sjson from 'secure-json-parse';
+
+export function parseJson(string) {
+  try {
+    return sjson(string);
+  } catch {
+    return '';
```

Comment:
this seems like an unusual default value...? I would expect `undefined` or `null`

Reply:
https://github.com/upleveled/next-js-example-winter-2023-vienna-austria/blob/ccaf7ec58f04066b612c7338707027caec1386ae/util/json.ts#L10